TLM/TLM/Manager/Impl/ExtVehicleManager.cs

```diff
@@ -910,7 +910,7 @@ public class ExtVehicleManager
 #endif
         }
 
-        private ExtVehicleType? DetermineVehicleTypeFromAIType(
+        public ExtVehicleType? DetermineVehicleTypeFromAIType(
```

Comment:
Could be `internal` too if you want to use it inside TrafficManager.dll

Reply:
Changed to `internal`